### DISCUSSION TITLE
Install on other linux distros that are not ubuntu like

### DISCUSSION THREAD

**Discussion by Fabick75** [phcode-dev/phoenix#1895] [General]
Hi, today I've discovered that brackets was substituted by this new project. This seems very interesting and then I've quickly take a look at your intaller script, because I don't use ubuntu like distros (usually I use fedora or salckware for my systems).
Then the installer script don't works on those two. I've tried to compile the project by myself using npm, but there are errors and npm usually is a nightmare to resolve conflicts. There is a change that in the near future you release an appimage of the project? I'm moving from Atom editor to this, because love to support new projects, but I don't want to fight with npm if it's not necessary, then my hope is that your team starts to release an appimage version in the near future.

**Comment by abose** [comment]
AppImage support is doable, but may not immediately come. 

Have you tried the manual installation steps for Linux outlined in our docs?

https://docs.phcode.dev/docs/Linux#manual-installation

**Reply by Fabick75** [reply]
I've tried, but even the npm modules building it is intertwined with ubuntu and debian. For example this script :

`node_modules/playwright-core/bin/reinstall_msedge_beta_linux.sh`

seem that all the project have explicit references to something that can be installed only if you're under a Debian or Ubuntu distros.

The calls to some installation steps running **npm install** give errors recalling terminal procedures, like this : `ReferenceError: fetch is not defined`
and I need to open the logs and discover that the error was :
`npm ERR! command sh -c gulp build`

then I've called manually : **gulp build** , and then other errors goes out and so on. I can swear that npm works well on other projects and can invoke the shell when needed.

The project seems very interesting to me, but if someone decide to develop something new under GNU/Linux, need to take in account that need to use a projecting path that take in accounts that people may use many different distros. Not only unbuntu, because all the Linux community know that Canonical put a lot of efforts to deeply personalize their distros (infact, usually I prefer to user Debian, when on server).

I can understand the effort profused in this new project, but at this point, it's better to wait for an AppImage version to test it.
Or maybe this can be build on a debian container? Maybe compiling all there and then try to recall him like a web service from outside, because if this works with browser and electron maybe expose as a service? I don't know. If you have a more detailed guide I will read it.
